### DISCUSSION TITLE
Welcome to The Little Learner and Malt Discussions!

### DISCUSSION THREAD

**Discussion by themetaschemer** [themetaschemer/malt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brrgrl** [comment]
Hi all,
thank you for the excellent repo!

I'm going through the little learner now and it's amazing!

**Comment by sinasibilsel** [comment]
should  (define line-xs [2.0 1.0 4.0 3.0]) and (define line-ys [1.8 1.2 2.3 3.3]) in chapter 1 of the little learner be
(define line-xs '[2.0 1.0 4.0 3.0]) and (define line-ys '[1.8 1.2 2.3 3.3]), respectively?

**Reply by themetaschemer** [reply]
The square brackets are a notation we use in the book to save space. To type it into code, use (tensor ...).

**Reply by sinasibilsel** [reply]
thank you.

**Comment by dfried00** [comment]
Also, we never use quote quasiquote and any other related
operator in either the punctuation or written out forms

**Reply by sinasibilsel** [reply]
thank you

**Comment by steve-ttt** [comment]
On page 26 the final version of line uses \\theta unicode 03B8 ? does racket allow the use of that, should I be using that in my code when writing it out or is it a short hand like described on page xxiii. The last para on that pages says it should be written out as a-hat,c-hat,big-theta,et al ; but then goes on to say unicode can be used in names of formals, functions and keywords? 

I wish the λ character was able to be use in other schemes and common lisp and I want that key on my keyboard (just sayin')

**Comment by steve-ttt** [comment]
page 64:frame 28 dot point 3 
Is that a typo should it read ((line (tensor 2.0 1.0 4.0 3.0)) (list 0.0 0.0)) 
not ((line (tensor 2.0 1.0 4.0 3.0)  (list 0.0 0.0))   
.                                                        ^{miss matched brackets}

**Comment by alexgian** [comment]
Hi and good to be here!

Currently working through the book and enjoying it a lot!
It's just what I wanted and at pretty much the level I wanted.  I am fairly comfortable in Scheme, I know a little about AD, but almost nothing of ML.

I am currently just at the end of Ch 9.  I found a little problem I got stuck on, which is what led me here.  I like the way the book has progressed so far, building and building on the gradient-descent function.  Reading ahead a bit, it just looks like it's getting better and better.  Looking forward to it.

Many thanks to all involved.
